### ISSUE TITLE
@phdthesis

### ISSUE THREAD

**Issue by yegor256** [yegor256/bibcop#88]
Exactly this should be there:

```
@phdthesis{citation-key,
    author = ,
    title = ,
    school = ,
    year = 
}
```

**Comment by tex-apprentice** [comment]
How come the version on CTAN right now - 2025/01/22 says:

```
Only @article, @book, @inproceedings, and @misc types of BibTEX entries are
allowed. Everything else, like @manual, @phdthesis, and many others are simply
prohibited. The mentioned four should be enough for everything.
```

Has this not been included in therelease yet?

**Comment by yegor256** [comment]
@tex-apprentice it's a typo in the documentation, fixed in d0e31d6
